src/main/java/uk/gov/hmcts/reform/authorisation/validators/AuthTokenValidator.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.authorisation.validators;
+
+interface AuthTokenValidator {
+
+    String BEARER = "Bearer ";
+
+    /**
+     * Validate bearer token.
+     * @param token Bearer token
+     */
+    void validate(String token);
+
+    /**
+     * Validate bearer token with optional roles.
+     * @param token Bearer token
+     * @param roles Roles
+     */
+    void validate(String token, String[] roles);
```

Comment:
a list would me nicer for everyone to use I think

Reply:
list of tokens or list of roles? or both?

Reply:
`roles`. List instead of an array.